cmd/pj-rehearse/main.go

```diff
@@ -141,3 +130,13 @@ func main() {
 		gracefulExit(o.noFail)
 	}
 }
+
+func createMasterWorktree(baseRSA string) error {
+	var out bytes.Buffer
+	cmd := exec.Command("git", "worktree", "add", "master", baseRSA)
```

Comment:
This assumes we run inside a clone and `o.candidatePath` was not provided leading somewhere else. `git` has a `-C` option that could be used here.

Also, I think this should explicitly return the path to the master worktree, so it can be used to infer paths to master config and master job configs.